lib/public/views/QcFlags/ActiveColumns/qcFlagsActiveColumns.js

```diff
@@ -13,10 +13,10 @@
 
 import { h } from '/js/src/index.js';
 import { formatTimestamp } from '../../../utilities/formatting/formatTimestamp.js';
-import { qcFlagColoredBadge } from '../common/qcFlagColoredBadge.js';
+import { qcFlagTypeColoredBadge } from '../common/qcFlagTypeColoredBadge.js';
 
 /**
- * List of active columns for a generic periods table
+ * Active columns configuration for QC flags pages
```

Comment:
Why pages?! This might perfectly be displayed in another page, the most you know is that it relates to QC flags table

Reply:
applied